cluster-provision/common-scripts/cnis-map.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 declare -A CNI_MANIFESTS
-CNI_MANIFESTS[1.17.0]="calico.yaml"
+CNI_MANIFESTS[1.17.0]="calico"
```

Comment:
This might cause problems at provision.sh and node01.sh on k8s-1.17/k8s-fedora-1.17
when applying this file with kubectl